src/wallet/rpcdump.cpp

```diff
@@ -217,6 +221,29 @@ Value importaddress(const Array& params, bool fHelp)
     return Value::null;
 }
 
+Value z_importwallet(const Array& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return Value::null;
+
+    if (fHelp || params.size() != 1)
+        throw runtime_error(
+            "z_importwallet \"filename\"\n"
```

Comment:
Originally they were included in the upstream calls dumpwallet and importwallet. Then after discussion with @nathan-at-least it was decided to separate them out into their own z_ commands.